docs/cli/index.rst

```diff
@@ -25,6 +25,10 @@ on-screen instructions:
 
     $ curl --proto '=https' --tlsv1.2 -sSf https://sh.edgedb.com | sh
 
+.. warning::
+
+    You should not run the CLI as root.
```

Comment:
This is generally a good idea, but it might work. If this is addressing what I think it is the major pain point is that people try to run a server as root which fails with an error and really should not be done.

@elprans may have more nuanced guidance for this.

Reply:
Here's the Slack discussion that inspired this for context: https://discord.com/channels/841451783728529451/1043930111528935555/1044655465977622619

Reply:
The fundamental issue here is that Postgres will refuse to run as root and so it is impossible to create instances as root.  Other operations are safe, so this should really be handled by the CLI in `create instance`.

Reply:
This also comes up when people try to run `edgedb-server-2` from the apt package as root. They usually don't understand what system-d is and that they don't actually need to start the server themselves.

Reply:
So i guess this bit should go in the bare metal deployment guide?

Reply:
@elprans @fmoor Thanks for your help on this. Check my understanding here: we should move this warning to the bare metal deployment guide, and the CLI itself needs to handle the scenario that the user tries to run `edgedb create instance` as root so there's no need to document that. Does that sound right?

Is the wording of the warning OK?

Reply:
> Does that sound right?

as I understand it, yes

> Is the wording of the warning OK?

The warning should say that `edgedb-server` can not be run as root. This is different than the CLI.